### DISCUSSION TITLE
grafana:toolkit Add command to lint (e.g. to tests)

### DISCUSSION THREAD

**Discussion by fr-ser** [grafana/grafana#46505] [Plugins]
The Grafana toolkit supports turning of linting for building (which I like). But there is no other way to run the linting (tests).

I would like to run tests and lint the code in a step without building. 

I think it would make sense to either add a command (e.g. plugin:lint) or an option (e.g. plugin.test --lint). Otherwise the only way to lint the code is to build it, which is counter intuitive to me.

What do you think?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
